xds/data/orca/v3/orca_load_report.proto

```diff
@@ -47,4 +47,10 @@ message OrcaLoadReport {
 
   // Application specific opaque metrics.
   map<string, double> named_metrics = 8;
+
+  // Application specific utilization expressed as a fraction of available
```

Comment:
Instead of saying "Application specifc", I suggest saying "Application-defined".

Also, I think it would be useful to give some examples of how this is expected to be used.  For example, the application might want to look at the utilization of multiple resources and then send the max of those values.

Reply:
Added an example, PTAL.

But this proto already uses the term "application specific" with `request_cost` and `named_metrics`, I guess it's more consistent to use the same term here?